### DISCUSSION TITLE
Please let me know what snacks(nibbles) and alcohol you recommend!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by miyazaki-koki** [NAIST-SE/AttractingContributors#89] [General]
I like to drink good alcohol drinks and eat snacks(nibbles).
Anything Ok. Please share it!!!!!!!!!!!!!!!!

https://github.com/miyazaki-koki/AttractingContributors

**Comment by sergiodl-naist** [comment]
Added Tequila Sunrise!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by iikyara** [comment]
Added Oden and Sake!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Comment by ellielo** [comment]
I don't recommend you any alcohol. I had my last glass of wine 9 months ago and i feel amazing!!! I started to do sports regulary, ordered myself vitamins and food supplements on [canadian pharmacy ](https://www.canadapharmacy.com/)site, drink lots of water, don't eat junks. You can't believe how my body changed. The transformation is unbelievable. I am definitely not going to back to "old" myself.
